server/CMakeLists.txt

```diff
@@ -1,11 +1,21 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.14)
 project(VenoraServer)
 
+# If missing this library, install with
+# `apt-get install libpq-dev postgresql-server-dev-all`
+find_package(PostgreSQL REQUIRED)
+
 add_subdirectory(logic)
 add_subdirectory(networking)
 add_subdirectory(database)
 
 add_executable(server server.c)
 
-target_link_libraries(server PRIVATE logic networking database)
-target_compile_options(server PRIVATE -g)
\ No newline at end of file
+target_include_directories(server PRIVATE ${PostgreSQL_INCLUDE_DIRS})
+
+target_link_libraries(server PRIVATE logic networking database ${PostgreSQL_LIBRARIES})
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
+
+target_compile_options(server PRIVATE -g -Wall)
```

Comment:
Yes these are redundant... leaving them in here for now because I may add some specific compile options for separate targets